### ISSUE TITLE
Clusters don't come up in scalability tests

### ISSUE THREAD

**Issue by oxddr** [kubernetes/kubernetes#85276]
**Which jobs are failing**:
gce-cos-1.17-scalability-100
gce-master-scale-correctness
gce-master-scale-performance

**Which test(s) are failing**:
Up

**Since when has it been failing**:
11/13

**Testgrid link**:
https://k8s-testgrid.appspot.com/sig-scalability-gce#gce-cos-1.17-scalability-100
https://k8s-testgrid.appspot.com/sig-scalability-gce#gce-master-scale-correctness
https://k8s-testgrid.appspot.com/sig-scalability-gce#gce-master-scale-performance

**Reason for failure**:
unknown yet

**Anything else we need to know**:

/sig scalability
/priority critical-urgent
/assign 
/cc @mborsz

**Comment by oxddr** [comment]
Based on systemd.log: bootstrapping script on master wasn't able to download K8s binaries:
```
Nov 14 06:06:17.460543 e2e-big-master systemd[1]: Failed to start Download and install k8s binaries and configurations.
```

**Comment by oxddr** [comment]
Requested binaries are available at the moment - I've just downloaded them from the same url (https://storage.googleapis.com/kubernetes-staging-dc037634a5/e2e-big-devel/kubernetes-server-linux-amd64.tar.gz).

**Comment by oxddr** [comment]
Should be fixed by https://github.com/kubernetes/kubernetes/pull/85220. Let's wait for the next run of gce100.

**Comment by liggitt** [comment]
#85220 merged, master and 1.17 scale 100 jobs are green

/close
